lib/static/css/main.css

```diff
@@ -713,6 +713,46 @@ pre code {
   padding: 1.25rem 1.5rem;
 }
 
+/* "Copy" code block button */
```

Comment:
Could we shift these CSS out into a separate file and load them as part of the siteConfig like how it's being done for the scripts?

It's not a good idea to put in `main.css` file otherwise it will be loaded even for people who opt out of using it. Since we make this feature opt-in in the script, we should make it opt-in in the styles too.

Reply:
Makes sense, importing via `stylesheets`. 

But I need to look at why the CSS is different now, will report back.